spice_ev/report.py

```diff
@@ -407,35 +407,35 @@ def aggregate_timeseries(scenario, gcID):
     if any(scenario.prices[gcID]):
         header.append("price [EUR/kWh]")
     # grid power
-    header.append("grid power [kW]")
-    # external loads
-    if hasExtLoads:
-        # external loads (e.g., building)
-        header.append("ext.load [kW]")
-    # feed-in
-    if any(scenario.feedInPower):
-        header.append("feed-in [kW]")
+    header.append("grid supply [kW]")
+    # fixed loads
+    if hasFixedLoads:
+        # fixed loads (e.g., building)
+        header.append("fixed load [kW]")
+    # local generation
+    if any(scenario.localGenerationPower):
+        header.append("local generation [kW]")
     # batteries
     if hasBatteries:
         header += ["battery power [kW]", "bat. stored energy [kWh]"]
     # flex
-    header += ["flex min [kW]", "flex base [kW]", "flex max [kW]"]
+    header += ["flex band min [kW]", "flex band base [kW]", "flex band max [kW]"]
```

Comment:
Can you change this for flex_min and flex_max in generate_schedule.py aswell? In the plot in line 730 (on dev - there are changes that need to be merged first)